werkzeug/utils.py

```diff
@@ -423,13 +423,7 @@ def import_string(import_name, silent=False):
             return sys.modules[import_name]
 
         module_name, obj_name = import_name.rsplit('.', 1)
-        try:
-            module = __import__(module_name, None, None, [obj_name])
-        except ImportError:
-            # support importing modules not yet set up by the parent module
-            # (or package for that matter)
-            module = import_string(module_name)
```

Comment:
The tests don't fail because of your PR, but I think this line is there for a reason (no tests there for it though?)